cli/azd/cmd/version.go

```diff
@@ -33,30 +35,39 @@ func newVersionFlags(cmd *cobra.Command, global *internal.GlobalCommandOptions)
 }
 
 type versionAction struct {
-	flags     *versionFlags
-	formatter output.Formatter
-	writer    io.Writer
-	console   input.Console
+	flags      *versionFlags
+	formatter  output.Formatter
+	writer     io.Writer
+	console    input.Console
+	llmManager llm.Manager
 }
 
 func newVersionAction(
 	flags *versionFlags,
 	formatter output.Formatter,
 	writer io.Writer,
 	console input.Console,
+	llmManager llm.Manager,
 ) actions.Action {
 	return &versionAction{
-		flags:     flags,
-		formatter: formatter,
-		writer:    writer,
-		console:   console,
+		flags:      flags,
+		formatter:  formatter,
+		writer:     writer,
+		console:    console,
+		llmManager: llmManager,
 	}
 }
 
 func (v *versionAction) Run(ctx context.Context) (*actions.ActionResult, error) {
 	switch v.formatter.Kind() {
 	case output.NoneFormat:
 		fmt.Fprintf(v.console.Handles().Stdout, "azd version %s\n", internal.Version)
+		time.Sleep(500 * time.Millisecond)
+		_, err := v.llmManager.Info(v.console.Handles().Stdout)
+		if err != nil {
+			return nil, fmt.Errorf("failed to get LLM info: %w", err)
+		}
+		fmt.Fprintf(v.console.Handles().Stdout, "\n")
```

Comment:
What does the sleep do here? Also noticed `azd version` now shows this error if I don't have the alpha flag set:
![image](https://github.com/user-attachments/assets/756ea941-7add-41a8-860c-ac31c2257680)